scripts/benchmarks/benchmark.py

```diff
@@ -257,7 +266,7 @@ def convert_keyvalue_arguments_to_list(args_dict: Dict):
             # otherwise if a regular argument
             if val is None:
                 warnings.warn(
-                    f"Argument '{arg}' is not a true/false argument andhad a 'None' value ",
+                    f"Argument '{arg}' is not a true/false argument andhad a 'None' value "\
```

Comment:
I dont understand why you need to replace with a `\` since it does the exact same thing